lib/travis/api/v3/services/crons/start.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  class Services::Crons::Start < Service
+
+    def run!
+      started = []
+
+      Models::Cron.all.each do |cron|
+        if cron.next_enqueuing <= Time.now
+          cron.start
+          started.push cron
+        end
+      end
```

Comment:
I'd love if you could move this into a query method. We try to encapsulate anything AR specific there.